pkg/czid/local.go

```diff
@@ -13,7 +16,7 @@ func IsInput(path string) bool {
 	return inputExp.MatchString(path)
 }
 
-var sampleNameExp = regexp.MustCompile(`(_R[12]|_R[12]_001)?\.(fasta|fa|fastq|fq)(\.gz)?$`)
+var sampleNameExp = regexp.MustCompile(`(_L\d\d\d)?(_R[12]|_R[12]_001)?\.(fasta|fa|fastq|fq)(\.gz)?$`)
```

Comment:
Nitpick: there can only be up to 8 lanes (L001 --> L008), so we could make this `_L00\d`